lib/pem.js

```diff
@@ -1002,7 +1015,9 @@ function spawnWrapper(params, tmpfiles, binary, callback) {
 
     spawnOpenSSL(params, binary, function(err, code, stdout, stderr) {
         files.forEach(function(file) {
-            fs.unlink(file.path);
+            fs.unlink(file.path, function (/*err*/) {
+                // TODO: Don't ignore unlink errors.
```

Comment:
I didn't fix these because doing so is a bit tricky with the synchronous `forEach` loop. This is still an improvement, as it prevents deprecation warnings on Node 7.